tests/cpp-tests/Classes/FileUtilsTest/FileUtilsTest.cpp

```diff
@@ -592,6 +593,127 @@ std::string TestWriteString::subtitle() const
     return "";
 }
 
+class CustomBuffer : public ResizableBuffer {};
+
+struct AlreadyExistsBuffer {};
+
+NS_CC_BEGIN
```

Comment:
`ResizableBufferAdapter` was defined in `cocs2d` namespace.
So specialized `ResizableBufferAdapter< AlreadyExistsBuffer >` sould defined in `cocs2d` namespace.